spec.md

```diff
@@ -253,14 +253,15 @@ that contains both context and data).
 
 ### event-type
 * Type: String
-* Description: Type of the event `data`. Producers can specify the format of
-  this, depending on their service. This enables the interpretation of `data`,
-  and can be used for routing, policy and more.
+* Description: Type of occurrence which has happened. Often this
+  property is used for routing, observability, policy enforcement, etc.
 * Constraints:
-  * REQUIRED
-  * MUST be a non-empty string
-* Examples:
-  * customer.created
+   * REQUIRED
+   * MUST be a non-empty string
+   * SHOULD be prefixed with a reverse-DNS name associated with the software
```

Comment:
@inlined I'm still unclear on this. Is it really the software that produces the event or the semantics of the event? I thought we wanted it to say `com.github.pull.create` even if the software component wasn't a github.com component, but it wanted receivers to act as if it did. Isn't the event type more about semantics of the event and less about where it came from - which would be `source` ?

Reply:
It should be semantic namespacing. Could change it to something like

> SHOULD be prefixed with a reverse-DNS name. The prefixed domain dictates
> the organization which defines the semantics of this event type.

Reply:
Works for me - thanks